python/pyspark/__init__.py

```diff
@@ -59,6 +59,8 @@ def since(version):
     indent_p = re.compile(r'\n( +)')
 
     def deco(f):
+        if not f.__doc__:
```

Comment:
This is a good idea, but can you please do it in a separate PR?  This is a broad change, so separating it out would be helpful (in case of conflicts, etc.).